x-pack/plugins/apm/public/components/shared/date_picker/apm_date_picker.tsx

```diff
@@ -26,11 +26,10 @@ export function ApmDatePicker() {
     rangeFrom,
     rangeTo,
     refreshPaused: refreshPausedFromUrl = 'true',
-    refreshInterval: refreshIntervalFromUrl = '0',
+    refreshInterval: refreshIntervalFromUrl = '60000',
```

Comment:
Wouldn't be better to have it in a variable and reuse it on the other components too?

```suggestion
    refreshInterval: refreshIntervalFromUrl = REFRESH_INTERVAL
```

Reply:
Fixed